build/docker/Dockerfile-bare

```diff
@@ -0,0 +1,40 @@
+# This dockerfile builds the zap stable release
+FROM ubuntu:16.04 as builder
+
+WORKDIR /zap
+
+RUN apt-get update && apt-get install -q -y --fix-missing \
+    wget \
+    curl \
+    xmlstarlet \
+    unzip && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Download and expand the latest stable release 
+RUN curl -s https://raw.githubusercontent.com/zaproxy/zap-admin/master/ZapVersions.xml | xmlstarlet sel -t -v //url |grep -i Linux | wget --content-disposition -i - -O - | tar zxv && \
+    cp -R ZAP*/* . &&  \
+    rm -R ZAP* 
+
+FROM java:openjdk-8-jdk-alpine
```

Comment:
java:* is deprecated, use openjdk:* (openjdk:8u121-jdk-alpine seems to work for now)

Reply:
That's a good one, I considered whether or not to choose openjdk, but I did not want to change too much in this commit - the original image used java.